releasenotes/notes/qpe-algorithms-primitives-3605bdfa5ab1bfef.yaml

```diff
@@ -0,0 +1,31 @@
+---
+features:
+  - |
+    Added options for quantum phase estimation algorithms
+    (:class:`~qiskit.algorithms.IterativePhaseEstimation`,
+    :class:`~qiskit.algorithms.PhaseEstimation`,
+    :class:`~qiskit.algorithms.HamiltonianPhaseEstimation`) to use
+    :class:`~qiskit.primitives.BaseSampler` primitive. See the example below.
+    Using :class:`~qiskit.utils.QuantumInstance` or :class:`~qiskit.providers.Backend` will now
+    issue a ``DeprecationWarning``.
```

Comment:
This is not quite right. The `quantum_instance` parameter (i.e. use of either of the two classes noted above) is now pending deprecation and will be deprecated in a future version.